src/components/validators/project-name-available-validator.js

```diff
@@ -28,6 +31,10 @@
 
             return deferred.promise;
           };
+
+          scope.$watch("organizationId", function() {
+            ngModel.$validate();
```

Comment:
Do we need to do this watch? What issue were you seeing with this? Also does this need to be validate async or will it automatically be handled?